WooCommerce/Classes/Tools/Logging/WooCrashLoggingStack.swift

```diff
@@ -129,6 +134,26 @@ class WCCrashLoggingDataProvider: CrashLoggingDataProvider {
             ServiceLocator.crashLogging.setNeedsDataRefresh()
         }
     }
+
+    // MARK: – Performance Monitoring
+
+    var performanceTracking: PerformanceTracking {
+        guard featureFlagService.isFeatureFlagEnabled(.performanceMonitoring) else {
+            return .disabled
+        }
+
+        return .enabled(
+            .init(
+                // FIXME: Is there a way to control this via feature flags?
+                sampler: { 0.1 },
```

Comment:
As long as our feature toggles are `Bool` only, the only way I can think to configure the sampler rate from the app is to have `.performanceMonitoringSampleRateTenPercent`, .performanceMonitoringSampleRateTwentyPercent`, etc. and a method to compute the numeric value to use here based on a rule such as "use the value corresponding to the highest feature flag enabled".

Eg.

```
performanceMonitoringSampleRateTenPercent = true
performanceMonitoringSampleRateTwentyPercent = false
performanceMonitoringSampleRateThirtyPercent = true

// => sample rate = 0.3
```

This sounds like a lot of feature toggles and something it would be easy to get wrong because of how easy it is to forget about updating all of the toggles.

Reply:
I feel like it would be useful to have a remote config for this sample rate, so that we can adjust it in production environment if we find there are too many events sent to Sentry and we have to pay unnecessary cost for it. But as a local configuration though, I feel like it's almost not worth the effort 😿

Reply:
> and something it would be easy to get wrong because of how easy it is to forget about updating all of the toggles.

My assessment on the risks of using a number of feature toggles to compute the sample rate might be greatly reduced once @hannahtinkler's feature toggles ship.

In particular, since the new feature toggles will be described in code, I can see a reverse of the computation logic described here in place where we give the backend a rate and it decides which toggles to serve as on and which off.